test/Lachain.ConsensusTest/BinaryBroadcastTest.cs

```diff
@@ -129,7 +129,7 @@ public void TestRandomFailures_7_2()
         }
 
         [Test]
-        [Repeat(10)]
+        [Repeat(1000)]
```

Comment:
decrease repeat count please (return it to old value),  1000 is too long for unit tests. We will add this trest to the external integratioin tests in future

Reply:
```suggestion
        [Repeat(10)]
```